app/views/user_mailer/send_confirmation_email.html.erb

```diff
@@ -0,0 +1,9 @@
+<div style="font-family: Arial, sans-serif; max-width: 600px; margin: 0 auto; padding: 20px; box-sizing: border-box;">
```

Comment:
please make it readable. wrap all css style inside `style` tag

Reply:
@kth-wtag Will be fixing all the view files in a separate PR.